### ISSUE TITLE
Recently played songs appears multiple times

### ISSUE THREAD

**Issue by ogbodotg** [Sangwan5688/BlackHole#36]
**Describe the bug**
Recently played songs appears the exact number of times a particular song is played. For instance if SONG1 is played 3 times, SONG1 appears 3 times under recently played songs. Since its still same song, one would expect it to appear only once irrespective of how many times a user plays it.

**To Reproduce**
1. Go to '...' Play a song, play it again and again
4. See error.. Check 'last session or recent' you'll see song appears number of times played.

**Expected behavior**
It is expected to appear just once since its the same song.

**Comment by Sangwan5688** [comment]
@ogbodotg This issue was already resolved in v1.6 or v1.7, I think. Please make sure you are using the latest version or commit. Still, I'm keeping the issue open until you confirm the same. Try using the latest version so that we can close the issue : )

**Comment by ogbodotg** [comment]
@sangwan5688 Yes, you are right. The latest version solved the issue.

Thank you

**Comment by Sangwan5688** [comment]
Cool :)

**Comment by ogbodotg** [comment]
Hi @Sangwan5688 I think this problem still exist when you play "recently played songs". I noticed it creates another duplicate of the song played.

**Comment by Sangwan5688** [comment]
I will check and confirm
